File: exercícios curtos/seleção de dificuldade/MainSelecaoDeDificuldade.kt
import java.util.*

fun main() {
    val scanner = Scanner(System.`in`)

    println("SELEÇÃO DE DIFICULDADE\n")

    var opcaoSelecionada = ""

    while (opcaoSelecionada == "") {
        println("ESCOLHA UMA OPÇÃO: fácil, médio e difícil")
        print("Sua escolha: ")
        val input = scanner.nextLine()
        println()

        val ehOpcaoValida = input == "fácil" || input == "médio" || input == "difícil"

        if (ehOpcaoValida) {
            opcaoSelecionada = input
        } else {
            println("OPÇÃO INVÁLIDA!!")
            println()
        }
    }

    println("Opção escolhida: $opcaoSelecionada")
}

File: exercícios curtos/seleção de dificuldade/main_selecao_de_dificuldade.py
print("SELEÇÃO DE DIFICULDADE\n")

opcao_selecionada = ""

while opcao_selecionada == "":
    print("ESCOLHA UMA OPÇÃO: fácil, médio e difícil")
    print("Sua escolha: ", end = "")
    entrada = input()
    print()

    eh_opcao_valida = entrada == "fácil" or entrada == "médio" or entrada == "difícil"

    if eh_opcao_valida:
        opcao_selecionada = entrada
    else:
        print("OPÇÃO INVÁLIDA!!")
        print()

print(f"Opção escolhida: {opcao_selecionada}")